tensorboard/plugins/hparams/tf_hparams_scatter_plot_matrix_plot/tf-hparams-scatter-plot-matrix-plot.html

```diff
@@ -641,7 +646,9 @@
         // Updates the internal state in response to a brush event in
         // the cell whose <g> element (in 'cells') is given by cellGNode
         function brushChanged(cellGNode) {
-          const brushSelection = d3.brushSelection(cellGNode);
+          // For some reason the closure compiler drops the argument when we
+          // write the call below as 'd3.brushSelection(cellGNode)'.
+          const brushSelection = d3["brushSelection"](cellGNode);
```

Comment:
No action required. As you have indicated, I couldn't find the reason why Closure deletes the argument from this method by inspecting both d3 code and playing around with defining type of this method. I am okay with this hack for time being.

Optional: Btw, although it is technically not in scope of this work, lines 628/639/642 have argument that are not used.

Reply:
Thanks! I removed the arguments.
Could you merge the PR, if you don't have any more comments?